src/server/middleware/executor/worker/package.json

```diff
@@ -3,8 +3,14 @@
   "main": "index.html",
   "dependencies": {
     "agentkeepalive": "=2.0.3",
+    "balanced-match": "^0.3.0",
+    "concat-map": "0.0.1",
+    "inflight": "^1.0.4",
+    "inherits": "^2.0.1",
     "mime": ">=1.2.11",
     "minimatch": ">=0.3.0",
+    "once": "^1.3.3",
+    "path-is-absolute": "^1.0.0",
     "pkginfo": ">=0.3.0",
```

Comment:
Are those packages used and weren't included before?
Or are they only used by the OS webkit?